lib/code_corps/map_utils.ex

```diff
@@ -8,18 +8,18 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  def stringify_keys(nil), do: nil
-  def stringify_keys(%DateTime{} = val), do: val
-  def stringify_keys(map = %{}) do
+  defp stringify_keys(nil), do: nil
+  defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
Having poked around in http://erlang.org/doc/man/erlang.html#is_atom-1, there pretty definitively isn't an `is_struct` and nothing jumps out at me as a good stand-in. I agree with @begedin, this needs to stay here for now.